app/components/ExportPackModal/ThirdStep.js

```diff
@@ -83,10 +83,11 @@ const ThirdStep = props => {
         ]
       },
       manifestType: 'minecraftModpack',
+      overrides: "overrides",
       manifestVersion: 1,
-      version: config.modpackVersion,
-      author: config.projectID ? data.primaryAuthorName : props.username,
-      projectID: config.projectID,
+      version: props.packVersion,
```

Comment:
I think we should default this to the existing one if it's available. For example, when exporting an already existing modpack, I would want to keep the original version instead of deciding a new one myself

Reply:
That isnt ideal since every Curse build will be called 1.0.0. Thats probally why it errors btw

Reply:
why what errors?
0.1.0 is just a default at the moment but the user is asked to input the next version they are going to deploy to.
Using the current version number would be wrong as well as it should be incremented up any only the pack auther would know if it should be major.minor.patch. if that makes sense.